packages/mongo-nav/src/types.ts

```diff
@@ -203,6 +203,7 @@ export interface CurrentProjectInterface extends ProjectInterface {
   alertsOpen: number;
   chartsActivated: boolean;
   status?: ProjectStatus;
+  useCNRegionsOnly: boolean;
```

Comment:
would `isCNRegionsOnly` make more sense?

Reply:
I agree that `isCNRegionsOnly` might make more sense but the field name on the project in the Atlas backend is called `useCNRegionsOnly` so I wanted to keep things consistent.